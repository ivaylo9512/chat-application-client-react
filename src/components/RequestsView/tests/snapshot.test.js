import RequestView from 'components/RequestsView/RequestsView';
import { shallow } from 'enzyme';

describe('RequestView snapshot tests', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <RequestView />
        )
    })
    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})