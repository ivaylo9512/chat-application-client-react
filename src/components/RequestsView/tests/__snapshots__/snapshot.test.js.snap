// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequestView snapshot tests should match snapshot 1`] = `
<styled.div>
  <RequestsList />
  <Pagination
    getData={[Function]}
    pagesPerSlide={5}
    selector={[Function]}
    setData={[Function]}
  />
</styled.div>
`;
